tests/UnifiedSpecTests/UnifiedTestRunner.php

```diff
@@ -235,6 +231,15 @@ private function doTestCase(stdClass $test, string $schemaVersion, ?array $runOn
         }
     }
 
+    /** @param list<array{failPoint: stdClass, server: Server}> $failPoints */
+    private function disableFailPoints(array $failPoints): void
+    {
+        foreach ($failPoints as ['failPoint' => $failPoint, 'server' => $server]) {
+            $operation = new DatabaseCommand('admin', ['configureFailPoint' => $failPoint, 'mode' => 'off']);
```

Comment:
```suggestion
            $operation = new DatabaseCommand('admin', ['configureFailPoint' => $failPoint->configureFailPoint, 'mode' => 'off']);
```

@alcaeus: This should resolve the runtime error you encountered.

That said, I came up with another PR that consolidates this logic into a ManagesFailPointsTrait, which is closer to what we had with FailPointObserver. These files are already quite complex, so I wanted to minimize additional clutter. I'll defer to you, though.

See: https://github.com/alcaeus/mongo-php-library/pull/1